runtimes/aleph-debian-11-python/init1.py

```diff
@@ -171,7 +182,9 @@ def setup_input_data(input_data: bytes):
             os.system("unzip -q /opt/input.zip -d /data")
 
 
-def setup_authorized_keys(authorized_keys: List[str]) -> None:
+def setup_authorized_keys(authorized_keys: Optional[List[str]]) -> None:
+    if authorized_keys is None:
+        return
```

Comment:
Same, seems unrelated to the PR. + I don't understand what's gained from passing an optional and having to check for None.

Reply:
Moved to https://github.com/aleph-im/aleph-vm/pull/408
This branch will need a rebase before continuing.